src/common/types.ts

```diff
@@ -0,0 +1,65 @@
+import EventStore from 'orbit-db-eventstore'
+
+export interface IMessage {
+  id: string
+  type: number
+  message: string
+  createdAt: number
+  channelId: string
+  signature: string
+  pubKey: string
+}
+
+export interface IRepo {
+  db: EventStore<IMessage>
+  eventsAttached: boolean
+}
+
+export interface IChannelInfo {
+  name: string
+  description: string
+  owner: string
+  timestamp: number
+  address: string
+  keys: { ivk?: string, sk?: string }
```

Comment:
We definitely don't need ivk and sk keys anymore.

Reply:
Yeah, but removing it would require changes in zbaylite. We'll do it in a separate branch